website/docs/r/cdn_frontdoor_firewall_policy.html.markdown

```diff
@@ -0,0 +1,267 @@
+
+->**NOTE:** The `Standard_AzureFrontDoor` Cdn Frontdoor firewall policy sku may contain `custom` rules only. The `Premium_AzureFrontDoor` Cdn Frontdoor firewall policy skus may contain both `custom` and `managed` rules.
+
+* `tags` - (Optional) A mapping of tags to assign to the Cdn Frontdoor firewall policy.
+
+---
+
+The `custom_rule` block supports the following:
+
+* `name` - (Required) Gets name of the resource that is unique within a policy. This name can be used to access the resource.
+
+* `action` - (Required) The action to perform when the rule is matched. Possible values are `Allow`, `Block`, `Log`, or `Redirect`.
+
+* `enabled` - (Optional) Is the rule is enabled or disabled? Defaults to `true`.
+
+* `priority` - (Required) The priority of the rule. Rules with a lower value will be evaluated before rules with a higher value. Defaults to `1`.
+
+* `type` - (Required) The type of rule. Possible values are `MatchRule` or `RateLimitRule`.
+
+* `match_condition` - (Required) One or more `match_condition` block defined below. Can support up to `10` `match_condition` blocks.
+
+* `rate_limit_duration_in_minutes` - (Optional) The rate limit duration in minutes. Defaults to `1`.
+
+* `rate_limit_threshold` - (Optional) The rate limit threshold. Defaults to `10`.
+
+---
+
+A `match_condition` block supports the following:
+
+* `match_variable` - (Required) The request variable to compare with. Possible values are `Cookies`, `PostArgs`, `QueryString`, `RemoteAddr`, `RequestBody`, `RequestHeader`, `RequestMethod`, `RequestUri`, or `SocketAddr`.
+
+* `match_values` - (Required) Up to `600` possible values to match. Limit is in total across all `match_condition` blocks and `match_values` arguments. String value itself can be up to `256` characters long.
+
+* `operator` - (Required) Comparison type to use for matching with the variable value. Possible values are `Any`, `BeginsWith`, `Contains`, `EndsWith`, `Equal`, `GeoMatch`, `GreaterThan`, `GreaterThanOrEqual`, `IPMatch`, `LessThan`, `LessThanOrEqual` or `RegEx`.
+
+* `selector` - (Optional) Match against a specific key if the `match_variable` is `QueryString`, `PostArgs`, `RequestHeader` or `Cookies`.
+
+* `negation_condition` - (Optional) Should the result of the condition be negated.
+
+* `transforms` - (Optional) Up to `5` transforms to apply. Possible values are `Lowercase`, `RemoveNulls`, `Trim`, `Uppercase`, `URLDecode` or`URLEncode`.
+
+---
+
+A `managed_rule` block supports the following:
+
+* `type` - (Required) The name of the managed rule to use with this resource.
+
+* `version` - (Required) The version on the managed rule to use with this resource.
+
+* `action` - (Optional) The action to perform when the managed rule is matched. Possible values are `Block`, `Log`, or `Redirect`.
```

Comment:
can you add what the default value is if this is omitted?

Reply:
Yep... fixed... BTW it `Block` 🚀